distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Regex.enso

```diff
@@ -1,59 +1,476 @@
 import project.Any.Any
+import project.Errors.Common.Syntax_Error
+import project.Data.Filter_Condition.Filter_Condition
+import project.Data.Map.Map
 import project.Data.Numbers.Integer
+from project.Data.Range.Extensions import all
```

Comment:
can we put these with the other `from` statements. I know we don't have a fixed order but I think imports first then forms in alphabetical order.

Reply:
Done